### ISSUE TITLE
Is it possible to change the lamp's minimum brightness level?

### ISSUE THREAD

**Issue by alex3d** [sbidy/pywizlight#65]
Is it possible to change the lamp's minimum brightness level?

I have Gauss downlight IoT 1600lm with a minimum brightness level of 10, and it is pretty bright for wakeup sequence.
I've seen `pwmRange` in UserConfig but it can't be changed from local control or from cloud Wiz Connect.

SystemConfig and UserConfig:
```
{'method': 'getSystemConfig', 'env': 'pro', 'result': {'mac': '', 'homeId': '', 'roomId': '', 'moduleName': 'ESP12_SHTW1C_01W', 'fwVersion': '1.21.2', 'groupId': 0, 'drvConf': [20, 1], 'ewf': [255, 0, 0, 255, 0, 0, 30], 'ewfHex': 'ff0000ff00001e', 'ping': 0}}
{'method': 'getUserConfig', 'env': 'pro', 'result': {'fadeIn': 500, 'fadeOut': 2000, 'fadeNight': False, 'dftDim': 100, 'pwmRange': [5, 100], 'whiteRange': [2700, 4000], 'extRange': [2700, 4000], 'po': False}}
```

@sbidy, sorry for hijacking this issue, but it seems the most relevant place to ask @vodovozovge one more question: What is the status of https://gitlab.com/wizlighting/wiz-local-control? Is it the official Wiz library other projects like pywizlight could take ideas from?

**Comment by vodovozovge** [comment]
>What is the status of https://gitlab.com/wizlighting/wiz-local-control? Is it the official Wiz library other projects like pywizlight could take ideas from?

@alex3d it's actively maintained and supported. We use it internally in some projects, for example in the tools used for manufacturing and testing. 
One can take a look there and have an idea about the protocol, method names, parameters, value ranges, etc. However, `pywizlight` is more advanced in certain features (for example CCT range detection).

> I've seen pwmRange in UserConfig but it can't be changed from local control or from cloud Wiz Connect.

PWM range is related to the hardware characteristics of the model and cannot be changed by the user.
Basically min-pwm specifies the minimum PWM value for the LEDs when they are not yet flickering and are able to render all the light modes correctly.

**Comment by alex3d** [comment]
Thanks for the quick answer.
I've had something like flickering in mind. But is it possible to change pwmRange without full spec compliance expectations at my own risk?

**Comment by sbidy** [comment]
@alex3d I'll close the issue because this cant be handled by the lib. But thank you for the contribution! 💙

**Comment by alex3d** [comment]
Thanks for handy library btw.
